packages/pixeloven-storybook/config/package.json

```diff
@@ -24,9 +24,9 @@
         "@pixeloven-webpack/types": "^6.1.0",
         "css-loader": "3.4.2",
         "deepmerge": "4.2.2",
-        "fork-ts-checker-webpack-plugin": "4.1.2",
-        "sass": "1.24.1",
-        "sass-loader": "8.0.0",
+        "fork-ts-checker-webpack-plugin": "4.1.6",
+        "sass": "1.26.2",
+        "sass-loader": "8.0.2",
```

Comment:
These three were slightly different from the webpack config. This just brings them inline